src/translations/en.json

```
@@ -1766,7 +1766,8 @@
           },
           "custom_integration": "custom integration",
           "error_from_custom_integration": "This error originated from a custom integration.",
-          "full_logs": "Full logs"
+          "full_logs": "Full logs",
```

```suggestion
          "show_full_logs": "Show full logs",
```
As we now have 2 different actions it might be good to add this to better separate them?
(if suggestion is accepted it needs more adjustments in the code)